### PR TITLE
Update debugging rails applications guide [ci skip]

### PR DESCRIPTION
**Pull request by ydakuka**

### CODE REVIEW

**Comment by kamipo** [comment]
https://github.com/rails/rails/pull/30349/files#diff-e7e88827a6782a9125492a6c1e17eeecR404 also.

**Comment by ydakuka** [comment]
fixed

File: guides/source/debugging_rails_applications.md [hunk 1]
```diff
@@ -401,7 +401,7 @@ To see the previous ten lines you should type `list-` (or `l-`).
    7      byebug
    8      @articles = Article.find_recent
    9
-   10      respond_to do |format|
+   10       respond_to do |format|
```
Comment (kamipo) [inline guides/source/debugging_rails_applications.md]: Remove a space.

**Comment by ydakuka** [comment]
Oops, sorry, fixed.

**Pull request merged**